src/snap-confine.apparmor.in

```diff
@@ -21,6 +21,8 @@
     /usr/lib/@{multiarch}/libseccomp.so* mr,
     /lib/@{multiarch}/libseccomp.so* mr,
 
+    @{PROC}/@{pid}/auxv r,
+
```

Comment:
I believe this was made by mvo later. I any case, I'm proposing this because I recall you asked for a review on the eventual upstream inclusion.